src/Platform/Microsoft.Testing.Platform/OutputDevice/Terminal/TerminalTestReporter.cs

```diff
@@ -51,7 +51,7 @@ internal sealed partial class TerminalTestReporter : IDisposable
     private bool _wasCancelled;
 
 #if NET7_0_OR_GREATER
-    [GeneratedRegex(@$"^   at ((?<code>.+) in (?<file>.+):line (?<line>\d+)|(?<code1>.+))$", RegexOptions.ExplicitCapture, 1000)]
+    [GeneratedRegex(@$"^\s*at ((?<code>.+) in (?<file>.+):line (?<line>\d+)|(?<code1>.+))$", RegexOptions.ExplicitCapture, 1000)]
```

Comment:
There should always be exactly 3 spaces in the stack frame, is this change needed or just done to make the regex more regexy?